README.md

```diff
@@ -1,143 +1,23 @@
-# Composer template for Drupal projects
+# Foreningsmentor
 
-[![Build Status](https://travis-ci.org/drupal-composer/drupal-project.svg?branch=8.x)](https://travis-ci.org/drupal-composer/drupal-project)
-
-This project template provides a starter kit for managing your site
-dependencies with [Composer](https://getcomposer.org/).
-
-If you want to know how to use it as replacement for
-[Drush Make](https://github.com/drush-ops/drush/blob/8.x/docs/make.md) visit
-the [Documentation on drupal.org](https://www.drupal.org/node/2471553).
-
-## Usage
-
-First you need to [install composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx).
-
-> Note: The instructions below refer to the [global composer installation](https://getcomposer.org/doc/00-intro.md#globally).
-You might need to replace `composer` with `php composer.phar` (or similar) 
-for your setup.
-
-After that you can create the project:
+## Setup for development
 
+Create settings.local.php
 ```
-composer create-project drupal-composer/drupal-project:8.x-dev some-dir --stability dev --no-interaction
+cp web/sites/default/docker.settings.local.php web/sites/default/settings.local.php
 ```
 
-With `composer require ...` you can download new dependencies to your 
-installation.
-
+Start docker containers
 ```
-cd some-dir
-composer require drupal/devel:~1.0
+itkdev-docker-compose up -d
```

Comment:
In most readme's we have write the docker without the dependency on itk-docker:

`docker-compose up --detach`
`docker-compose exec phpfpm composer install`

Reply:
Changed